### DISCUSSION TITLE
SMART  is possible?

### DISCUSSION THREAD

**Discussion by Iberid1974** [amidaware/tacticalrmm#2055] [General]
good everyone!
how are they? I hope very well.
I wanted to ask if anyone has experience verifying the status of disks using a SMART check. Thank you very much in advance.

Example:
![images](https://github.com/user-attachments/assets/fd520963-5f01-4e4f-9aa0-228d708e0555)

**Comment by P6g9YHK6** [comment]
https://github.com/Lifailon/CrystalDisk-Cli
CD is very good but never tried the cli maybe it's worth a look.

**Reply by Iberid1974** [reply]
Thanks , now I'll read it and see if I can move forward.

**Comment by silversword411** [comment]
There's lots of scripts in the library for this

**Reply by Iberid1974** [reply]
yes? which? Thanks

**Reply by silversword411** [reply]
Search for it
https://github.com/search?q=repo%3Aamidaware%2Fcommunity-scripts%20smart&type=code
